components/rating/index.css

```diff
@@ -74,37 +122,161 @@ governing permissions and limitations under the License.
   position: relative;
   background-size: contain;
   background-repeat: no-repeat;
-  inline-size: var(--spectrum-rating-icon-width);
-  block-size: var(--spectrum-icon-star-height);
-  padding: var(--spectrum-rating-icon-padding-y) 0;
+  inline-size: var(--mod-rating-icon-width, var(--spectrum-rating-icon-width));
+  /* TODO */
+  /*block-size: var(--mod-rating-icon-height, var(--spectrum-icon-star-height));*/
+  padding: var(--mod-rating-icon-spacing-vertical-top, var(--spectrum-rating-icon-spacing-vertical-top));
```

Comment:
I think this padding may be causing the spacing between the icons to be too wide. Let's delete this line.